database.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+SQLALCHEMY_DATABASE_URI = 'sqlite:///library.db'
```

Comment:
The `SQLALCHEMY_DATABASE_URI` is set to use SQLite. Ensure that this is the intended database for your application. If you plan to use a different database in production, update this URI accordingly.